### ISSUE TITLE
Add aws-cli to default console for ecs enable image

### ISSUE THREAD

**Issue by bseng** [rancher/os#1652]
**RancherOS Version: (ros os version)**
V0.8.0

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**
aws

Just a suggestion is it possible to add aws-cli to default console for ecs enable image so anyone using can use aws-cli at runcmd without needing to switch to ubuntu console and apt-get install aws-cli.

Thanks.

**Comment by SvenDowideit** [comment]
not everyone that uses RancherOS is using aws - so we don't want to add AWS specific things to the default console.

personally, I think you should probably do this by using `docker run` - using something like https://store.docker.com/community/images/cgswong/aws

**Comment by bseng** [comment]
I understand not everyone uses aws so that's why I suggested to have it available in ecs enable images https://docs.rancher.com/os/amazon-ecs/

for things like attaching ebs volumes before starting docker I can't really use a docker image for that.

**Comment by joshwget** [comment]
@SvenDowideit Something wonka-like would probably work well here.

**Comment by SvenDowideit** [comment]
@bseng when you say "docker", you mean user-docker? - ie, you can use a docker container, running in system-docker before the user-docker is started.

(I'm asking because there is a need for things like this, but we need to work out the best approach)

**Comment by bseng** [comment]
yes I mean user-docker, but I guess it also depends on when will a system-docker image available in the booting process.

if the docker image is available before runcmd and I can use aws-cli at runcmd that I'll be great.

It will be good if the image is already packaged in the ecs image then the start-up time will be shortened.
